### DISCUSSION TITLE
Show filename in note view top bar (Android)

### DISCUSSION THREAD

**Discussion by craigfurman** [GitJournal/GitJournal#869] [Feature Requests]
When viewing a note in the Android app, the note filename doesn't appear to be visible anywhere on screen. I rely on filenames as de facto titles in most notes when I view/edit them in Obsidian for desktop, so it'd be really convenient to be able to view the filename in Android's note view.

There's currently not a lot of text in the top bar so the filename be a nice addition. What do you think?

**Comment by vHanda** [comment]
I think it'll make more sense to just change the 'title bar' in the top to the 'filename', this way it would be more compatible with Obsidian.

I just don't know how to do this in a non-breaking way, as the current title refers to the 'h1'.

**Comment by craigfurman** [comment]
Sure, that sounds like it would do the trick!

**Comment by craigfurman** [comment]
@vHanda WDYT of showing the filename in the title bar if no h1 is present? That would be non-breaking. I could send a PR if it's wanted?

**Comment by vHanda** [comment]
> @vHanda WDYT of showing the filename in the title bar if no h1 is present? That would be non-breaking. I could send a PR if it's wanted?

Sorry about the delay. I've been swamped with some consulting projects.

Yes, please. Go ahead.

**Comment by eichin** [comment]
Found this while using android gitjournal to poke at a logseq repo; logseq pages don't have a `#` title at all, and usually start with a bullet list, so the filename is the only hint.  So @craigfurman 's approach would handle that perfectly.
